Client/Frontend/Theme/ThemeManager.swift

```diff
@@ -10,6 +10,11 @@ enum ThemeManagerPrefs: String {
     case themeName = "prefKeyThemeName"
 }
 
+enum SystemThemeNames: String {
```

Comment:
Not needed, there is already UIUserInterfaceStyle which is a enum with these values (light, dark)